app-crypt/paperkey/paperkey-1.4.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Tool to extract secret bits from a secret key to be printed on paper"
+DESCRIPTION="Paperkey extracts the secret bits from OpenPGP keys, to be printed \
+on paper as a form of backup and allows it to be reintroduced to reconstruct the \
+secret key if needed."
+HOMEPAGE="http://www.jabberwocky.com/software/paperkey/"
+COPYRIGHT="2007-2016 David Shaw"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.jabberwocky.com/software/paperkey/paperkey-$portVersion.tar.gz"
+CHECKSUM_SHA256="e12bb0ec835127d12a922a8d60b3dfdb3ca8ee60bb5b4d15ae4cea85bbcf336f"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	paperkey$secondaryArchSuffix = $portVersion
+	cmd:paperkey$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	runConfigure ./configure
```

Comment:
It looks like beecrypt only needs libcrypto. This means we just need to add:
* in REQUIRES:
```
	lib:libcrypto$secondaryArchSuffix
```
* and in BUILD_REQUIRES:
```
	devel:libcrypto$secondaryArchSuffix
```

But you also need to change the call to make in BUILD to:
```
	make $jobArgs LIBS="-lcrypto"
```